mappings/net/minecraft/world/gen/feature/Feature.mapping

```diff
@@ -28,7 +28,7 @@ CLASS net/minecraft/class_3031 net/minecraft/world/gen/feature/Feature
 	METHOD method_13150 register (Ljava/lang/String;Lnet/minecraft/class_3031;)Lnet/minecraft/class_3031;
 		ARG 0 name
 		ARG 1 feature
-	METHOD method_13151 generate (Lnet/minecraft/class_1936;Lnet/minecraft/class_2794;Ljava/util/Random;Lnet/minecraft/class_2338;Lnet/minecraft/class_3037;)Z
+	METHOD method_13151 place (Lnet/minecraft/class_1936;Lnet/minecraft/class_2794;Ljava/util/Random;Lnet/minecraft/class_2338;Lnet/minecraft/class_3037;)Z
```

Comment:
Why rename this?

Reply:
@SuperCoder7979 I wish to hear your opinion on this.

Reply:
Because we are "Placing" things into the world and not generating things into the world

Reply:
Well, this refactor affects some mods (frequently used by worldgen mods), so I guess... I will ask @valoeghese to take a peek as well

Reply:
Woah well lets not break that then. I changed it back

https://github.com/FabricMC/yarn/pull/1218/commits/d8896671c90cc430a5be3dc124258a6211af1a50